### PR TITLE
Fixed and extended the export

### PR DESCRIPTION
**Pull request by ms-syscon**
- fixed lastedit date in export
- added columns Project, Priority, lastedit and assignedto to export

### CODE REVIEW

**Comment by peterdd** [comment]
Please do not mix several topics in one pull request.

Solving one issue with 1 PR makes review easier.

I try to review stuff this month.

File: includes/class.backend.php [hunk 1]
```diff
@@ -310,8 +310,11 @@ public static function add_comment($task, $comment_text, $time = null)
         }
 
 	if ($conf['general']['syntax_plugin'] != 'dokuwiki') {
+    $properties = array();
+    if (isSet($conf['html']['allowed_css_properties']))
```
[the remark this answers is not in the document]
Comment (ms-syscon) [reply includes/class.backend.php:314]: Do you mean isset() in lower case?

**Comment by ms-syscon** [comment]
> Please do not mix several topics in one pull request.
> 
> Solving one issue with 1 PR makes review easier.
> 
> I try to review stuff this month.

I'm new on pull request. How to make it right? One branch for each issue?

**Comment by ms-syscon** [comment]
I will open a new one from a branch

**Pull request closed without merging**